test/cmd/get.sh

```diff
@@ -491,3 +491,26 @@ __EOF__
   set +o nounset
   set +o errexit
 }
+
+run_get_chunk_size_with_label_selector_tests() {
```

Comment:
This is not the right place to test that change, it should go in one of the list / pagination tests. Should be an integration test.

Reply:
Done.Function test and integration test have been updated.